js/views/chatAppVw.js

```diff
@@ -0,0 +1,267 @@
+var __ = require('underscore'),
+    Backbone = require('backbone'),
+    $ = require('jquery'),
+    chatCollection = require('../collections/chatCl'),
+    chatView = require('./chatVw'),
+    chat = require('../models/chatMd'),
+    chatMessageCollection = require('../collections/chatMessageCl'),
+    chatMessageView = require('./chatMessageVw'),
+    chatMessage = require('../models/chatMessageMd'),
+    loadTemplate = require('../utils/loadTemplate');
+Backbone.$ = $;
+
+module.exports = Backbone.View.extend({
+
+  el: '#sideBar',
+
+  events: {
+    'click .js-newChat': 'newChat',
+    'click .js-closeChat': 'closeChat',
+    'click .js-closeConversation': 'closeConversation',
+    'click .js-chatSearch': 'chatSearch',
+    'keyup .js-chatMessage': 'sendChat'
+  },
+
+  initialize: function(options){
+    var self = this;
+    this.options = options || {};
+    this.parentEl = $(options.parentEl);
+    this.socketView = options.socketView;
+
+    // Render chat list items
+    this.listWrapper = $('<div class="border0 custCol-border-secondary flexRow"></div>');
+    this.chats = new chatCollection();
+    var model = this.options.model;
+
+    this.subViews = [];
+    this.subViewsChat = [];
+    this.render();
+
+    this.chats.url = model.get('serverUrl') + "get_chat_conversations";
+    this.chats.fetch({
```

Comment:
This has to be moved to the callback of render. Otherwise it will try to fire renderChat before the page has rendered. 

Anything that manipulates the DOM has to happen after render() is called.